src/modules/scraper/adapter/messaging/BlocksEventsConsumer.ts

```diff
@@ -77,6 +94,7 @@ export class BlocksEventsConsumer {
     }));
     await this.scraperQueuesService.publishMessagesBulk<FillEventsQueueMessage>(ScraperQueue.FillEvents, fillMessages);
 
+    await this.insertRawSpeedUpEvents(speedUpEvents);
```

Comment:
after the speedup events are fetched for a particular block range, save them in the DB